mcs/tools/mono-api-html/ConstructorComparer.cs

```diff
@@ -133,10 +133,15 @@ public override string GetDescription (XElement e)
 			if (parameters != null) {
 				var list = new List<string> ();
 				foreach (var p in parameters.Elements ("parameter")) {
-					var pTypeName   = p.GetTypeName ("type");
-					list.Add (State.IgnoreParameterNameChanges
-						? pTypeName
-						: pTypeName + " " + p.GetAttribute ("name"));
+					var param = p.GetTypeName ("type");
+					if (!State.IgnoreParameterNameChanges)
+						param += " " + p.GetAttribute ("name");
+
+					var direction = p.GetAttribute ("direction");
+					if (direction?.Length > 0)
```

Comment:
I'd use `!String.IsNullOrEmpty (direction)` here but that's just personal preference